File: src/JavaScript/easy/array/problem2108.js
/**
 * @param {string[]} words
 * @return {string}
 */

var firstPalindrome = function(words) {
    for(let word of words){
        if(word == word.split("").reverse().join("")){
            return console.log(word);
        }
    }
    return ""
};

let words = ["abc","car","ada","racecar","cool"]

firstPalindrome(words)

File: src/Python/easy/array/problem2108.py
from typing import List

class Solution:
    def firstPalindrome(self, words: List[str]) -> str:
        for word in words:
            if word == "".join(list(word)[::-1]):
                return word
        return ""


words = ["abc","car","ada","racecar","cool"]

solution = Solution()
solution.firstPalindrome(words)